### PR TITLE
Adjust validation for subject name - do not allow null value and empty value

### PR DESCRIPTION
**Pull request by mslowiak**
Adjusted validation in three places:
- `EmbeddingController`
- `SubjectController`
- `SubjectDto`

Replaced NotNull with NotBlank annotation (which checks it the value is the null or empty)

Performed some manual tests to check behaviour after fix.

Closes: #617

### CODE REVIEW

**Comment by mslowiak** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mslowiak** [comment]
recheckcla

**Comment by mslowiak** [comment]
@pospielov 
Would you mind taking a look at this? :)

File: java/api/src/main/java/com/exadel/frs/core/trainservice/controller/EmbeddingController.java [hunk 1]
```diff
@@ -51,7 +52,7 @@
     @PostMapping
     public EmbeddingDto addEmbedding(
             @ApiParam(value = IMAGE_WITH_ONE_FACE_DESC, required = true) @RequestParam final MultipartFile file,
-            @ApiParam(value = SUBJECT_DESC, required = true) @RequestParam(SUBJECT) final String subjectName,
+            @ApiParam(value = SUBJECT_DESC, required = true) @Valid @NotBlank(message = "Subject name is empty") @RequestParam(SUBJECT) final String subjectName,
```
Comment (pospielov) [inline java/api/src/main/java/com/exadel/frs/core/trainservice/controller/EmbeddingController.java:55]: Please move the message here:
com.exadel.frs.core.trainservice.system.global.Constants

**Review by pospielov — CHANGES_REQUESTED** [review]
The validation doesn't work
Please check this:
https://stackoverflow.com/a/62386664
I believe it should start working after you add this annotation.

https://documenter.getpostman.com/view/17578263/UUxzAnde#66d07d92-f7b7-4e4c-a993-5ae3f487ed8e
here is the postman collection to check if it works

**Comment by mslowiak** [comment]
@pospielov I fixed the review comments

**Review by pospielov — APPROVED** [review]

**Comment by pospielov** [comment]
Thanks for your contribution! :)

**Pull request merged**